.buildkite/pipeline.yml

```diff
@@ -57,15 +57,15 @@ steps:
       queue: "{{ matrix }}"
     env:
       # See https://playwright.dev/docs/ci#debugging-browser-launches
-      DEBUG: 'pw:browser'
+      DEBUG: "pw:browser"
     matrix:
       - mac
       #- windows
 
-  - group: "📦 Build for Mac"
-    key: 'dev-mac'
+  - group: ":package: Build for Mac"
+    key: dev-mac
     steps:
-      - label: '🔨 Mac Dev Build - {{matrix}}'
+      - label: ":hammer: Mac Dev Build - {{matrix}}"
```

Comment:
You might want to add a custom `notify: { github_commit_status: {context: … } }` commit status to go with that change without the GitHub PR checks to be affected…

Reply:
Done.

I also think we agreed to have `notify.github_custom_status` as part of our recommendations, right? I had forgotten about it here.